### ISSUE TITLE
Fishing port problem

### ISSUE THREAD

**Issue by RACING799** [dymanoid/RealTime#263]
The operation of a fishing port has encountered a very strange problem

After the game is loaded, because the game time is night, the fishing boats that have already sailed to the sea return to the port and stop sailing.

The strange problem was that I built a new fishing port at night and it was miraculously working 

Did the author think of a possible cause of the problem?

![78](https://user-images.githubusercontent.com/62243313/77983929-cc3ff200-7342-11ea-9b73-ba3e021719a0.PNG)
![155](https://user-images.githubusercontent.com/62243313/77983933-cf3ae280-7342-11ea-886a-fb0d16a4d9ff.PNG)

**Comment by dymanoid** [comment]
Yes, the fishing industry has only a single work shift in Real Time, and this is the day shift. In the night, the fishing industry is closed.

When you place a new harbor in the night, the game first thinks that the building is active, so a boat is sent out for fishing. Then, in a couple of minutes, the game sees that the building should actually be inactive, and all fishing boats return to the harbor.

This is a little bit strange, I agree. I will investigate if there is something I can do with it.
